sass/typography/presets/_fluent.scss

```diff
@@ -88,6 +98,7 @@ $type-scale: (
     button: type-style(
         $font-size: rem(14px),
         $font-weight: 400,
+        $letter-spacing: 0,
```

Comment:
```suggestion
        $letter-spacing: rem(.75px),
```

Reply:
In the master branch the letter spacing is 0, isn't it?
<img width="274" alt="Screenshot 2022-07-30 at 17 07 06" src="https://user-images.githubusercontent.com/45598235/181918125-7a58286c-f430-47d9-8656-a0fbd416505c.png">
<img width="296" alt="Screenshot 2022-07-30 at 17 08 20" src="https://user-images.githubusercontent.com/45598235/181918126-da28c2f8-b820-40bf-8484-1f7a4e89d3c1.png">